packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -431,6 +464,12 @@ export class FileNavigatorContribution extends AbstractViewContribution<FileNavi
             keybinding: 'ctrlcmd+i',
             context: NavigatorKeybindingContexts.navigatorActive
         });
+
+        registry.registerKeybinding({
+            command: FileNavigatorCommands.COPY_RELATIVE_FILE_PATH.id,
+            keybinding: isWindows ? 'ctrl+shift+alt+c' : 'ctrlcmd+shift+alt+c',
```

Comment:
I thought `ctrlcmd` is `ctrl` on Window and Linux, hence I do not understand the `isWindows` check. Cannot we have `'ctrlcmd+shift+alt+c'`  for all platforms?

Reply:
VS Code uses a keychord for this on Windows. Can we do the same?
![image](https://user-images.githubusercontent.com/1405703/85850118-75648000-b7ac-11ea-8932-bb77d7a57401.png)

The copied paths have Windows separator (`\`) in VS Code:
```
packages\core\src\node\env-variables\env-variables-server.ts
packages\core\src\node\env-variables\index.ts
packages\core\src\node\messaging
packages\core\src\node\application-server.ts
packages\core\src\node\backend-application.ts
packages\core\src\node\backend-application-config-provider.ts
packages\core\src\node\backend-application-module.ts
```

With this PR, the copied relative paths look like this:
```
packages/core/src/common/selection-command-handler.ts
packages/core/src/common/resource.ts
packages/core/src/common/reference.ts
packages/core/src/common/reference.spec.ts
packages/core/src/common/quick-pick-service.ts
packages/core/src/common/quick-open-service.ts
packages/core/src/common/quick-open-model.ts
packages/core/src/common/promise-util.ts
```

Reply:
@kittaakos thank you for the review, unfortunately I don't have a windows machine to test when preparing the pull-request so I appreciate you testing on windows. Out of curiosity, for the file paths do you have the same issue with the `copy path` command, I based my implementation on it.

Reply:
Yes.
Theia: /c:/Users/kittaakos/dev/git/theia/packages/core/src/common/menu.spec.ts
VS Code: C:\Users\kittaakos\dev\git\theia\packages\core\src\common\menu.spec.ts

Reply:
@kittaakos I've updated the keybinding on windows to use the chord.
I haven't included updates to the path (separator), since looking at the discussions of #8098 it looks like we would need a generic solution which updates all paths on windows properly.